api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -83,4 +83,8 @@ message AWSLambdaConfig {
     google.protobuf.Duration timeout = 3;
   }
 
+    // Send downstream path and method as `x-envoy-original-path` and `x-envoy-original-method`
+    // headers on the request to AWS lambda. 
+    // Defaults to false.
```

Comment:
this looks good. just wanted to call out the sentiment earlier behind my earlier comment to rewrite this documentation here.

in general it's good to write our proto docs with the following guidance/style: "start with what it does, followed by its default value, and then any callouts or notices for devs/users"